### PR TITLE
fix plugin installer

### PR DESCRIPTION
**Pull request by van800**
Do not do anything about old plugin of 2 files.
Handle only Unity3dRider.cs and new dll format.

### CODE REVIEW

File: resharper/src/resharper-unity/Rider/UnityPluginDetector.cs [hunk 1]
```diff
@@ -17,11 +15,9 @@ public class UnityPluginDetector
         
         private readonly ISolution mySolution;
         private readonly ILogger myLogger;
-        private static readonly string[] ourPluginFilesV180 = {"RiderAssetPostprocessor.cs", "RiderPlugin.cs"};
```
Comment (IvanPashchenko) [inline resharper/src/resharper-unity/Rider/UnityPluginDetector.cs]: Am I right that the last time we used this was pre-release Rider, so there were no stable Rider versions with this plugin variation bundled?

File: resharper/src/resharper-unity/Rider/UnityPluginDetector.cs [hunk 2]
```diff
@@ -193,69 +129,23 @@ private InstallationInfo ExistingInstallation(List<FileSystemPath> pluginFiles)
...
-            }
             
             var pluginDir = parentDirs[0];
-            var filenames = pluginFiles.Select(f => f.Name).ToList();
 
-            // v1.9.0+
             if (pluginFiles.Count == 1)
             {
-                if (pluginFiles[0].Name == MergedPluginFile)
+                if (pluginFiles[0].Name == PluginDllFile)
```
Comment (IvanPashchenko) [inline resharper/src/resharper-unity/Rider/UnityPluginDetector.cs]: It looks like we won't be able to upgrade from `Unity3DRider.cs` to `.dll` this way.

File: resharper/src/resharper-unity/Rider/UnityPluginDetector.cs [hunk 3]
```diff
@@ -193,69 +129,23 @@ private InstallationInfo ExistingInstallation(List<FileSystemPath> pluginFiles)
...
                 {
-                    var version = GetVersionFromFile(pluginFiles[0]);
+                    var version = new Version(FileVersionInfo.GetVersionInfo(pluginFiles[0].FullPath).FileVersion);
                     return new InstallationInfo(version != ZeroVersion, pluginDir, pluginFiles, version);
                 }
                 
                 myLogger.Warn("One file found, but filename is not the same as v1.9.0+");
```
Comment (IvanPashchenko) [inline resharper/src/resharper-unity/Rider/UnityPluginDetector.cs]: This error message is obsolete.

File: resharper/src/resharper-unity/Rider/UnityPluginDetector.cs (hunk 1 above)
Comment (van800) [reply resharper/src/resharper-unity/Rider/UnityPluginDetector.cs]: yes

**Pull request merged**